### ISSUE TITLE
Vibrato Class

### ISSUE THREAD

**Issue by RitheshKumar** [GTCMT/MUSIC-8903-2016#3]
in resetInstance()

there is no nullptr check on the members before calling delete on them

**Comment by amrutavidwans** [comment]
Yes, because of this the host aborts giving memory access error while implementing in plugin (though the build succeeds in Xcode).

**Comment by alexanderlerch** [comment]
good catch. You guys can fix it, right?

**Comment by RitheshKumar** [comment]
Yes, a simple nullptr check should suffice. Thanks to amruta for pointing it out!
